services/web/server/src/simcore_service_webserver/errors.py

```diff
@@ -4,5 +4,7 @@
 
 
 class WebServerBaseError(OsparcErrorMixin, Exception):
+    msg_template = "Unexpected error in web-server"
+
     def __init__(self, **ctx: Any) -> None:
         super().__init__(**ctx)
```

Comment:
```suggestion
```

Reply:
@giancarloromeo is there a difference? sorry, i cannot see it :-)

Reply:
Not yet, I suggested to remove the constructor

Reply:
hahah I did not notice the (obvious) color! :-D